docs/manuals/core/config/scenario-config.md

```diff
@@ -44,18 +44,20 @@ Behind the scenes, a pipeline configuration is created. Its id will be the scena
     ```python linenums="1"
     from taipy import Config
 
-    scenario_config = Config.configure_scenario_from_tasks("multiply_scenario", [task_config], pipeline_id="multiply_pipeline")
+    scenario_config = Config.configure_scenario_from_tasks(
+        "multiply_scenario", [task_config], pipeline_id="multiply_pipeline")
     ```
 
-# Scenario configuration with Cycle
+# Using Cycles
```

Comment:
You did not use capitals in the previous similar sections (task, pipeline...). Why here?